src/components/table-toolbar/KeycloakDataTable.tsx

```diff
@@ -203,7 +203,18 @@ export function KeycloakDataTable<T>({
   const [key, setKey] = useState(0);
   const refresh = () => setKey(new Date().getTime());
 
+  const renderCell = (columns: (Field<T> | DetailField<T>)[], value: T) => {
+    return columns.map((col) => {
+      if (col.cellRenderer) {
+        return { title: col.cellRenderer(value) };
+      }
+      return _.get(value, col.name);
+    });
+  };
```

Comment:
Since arrow functions return the value from the last statement this code could be shortened a bit:

```suggestion
  const renderCell = (columns: (Field<T> | DetailField<T>)[], value: T) =>
    columns.map((col) => {
      if (col.cellRenderer) {
        return { title: col.cellRenderer(value) };
      }
      return _.get(value, col.name);
    });
```

You could make it even more compact if you use a ternary: 

```suggestion
  const renderCell = (columns: (Field<T> | DetailField<T>)[], value: T) =>
    columns.map((col) =>
      col.cellRenderer
        ? { title: col.cellRenderer(value) }
        : _.get(value, col.name)
    );
```